Makefile

```diff
@@ -13,9 +13,12 @@ SCENARIO := $(if $(SCENARIO),$(SCENARIO),"with_rag")
 images: ## Build container images
 	scripts/build-container.sh
 
-install-tools: ## Install required utilities/tools
+install-tools:	install-woke ## Install required utilities/tools
 	@command -v pdm > /dev/null || { echo >&2 "pdm is not installed. Installing..."; pip install pdm; }
 
+install-woke: ## Install woke, required for Inclusive Naming scan
+	@command -v ./woke > /dev/null || { echo >&2 "woke is not installed. Installing..."; curl -sSfL https://git.io/getwoke | bash -s -- -b ./; }
```

Comment:
did you add `woke` to your .gitignore?

Reply:
nice catch
it was added locally but forgot to include it in the commit